initrd-progs/0initrd/init

```diff
@@ -969,6 +977,10 @@ find_onepupdrv "$A_PART" "$A_BP_FN" "$A_DEF_FN" "a"
 [ "$ONE_FN" ] && ADRV="$ONE_PART,$ONE_FS,$ONE_FN"
 [ "$ADRV" ] && { LOADMSG="adrv"; setup_onepupdrv "$ADRV" "a" "p"; }
 
+find_onepupdrv "$N_PART" "$N_BP_FN" "$N_DEF_FN" "a"
+[ "$ONE_FN" ] && NDRV="$ONE_PART,$ONE_FS,$ONE_FN"
+[ "$NDRV" ] && { LOADMSG="ndrv"; setup_onepupdrv "$NDRV" "n" "p"; }
```

Comment:
I was thinking whether the third arg should have been added or not before this PR but at the end of the day it doesn't matter because NLS files won't cover anything anyway.

Reply:
Why does it need it's own ".sfs" file? (Why not include it in puppy....sfs?)
Why does it need to be "above" the 'puppy...sfs'?
Could it be appended just like an extra-sfs? (It could be done fairly easily in 'rc.sysinit'.)
If it must be a Puppy system '.sfs', where is the definition of the filename?
Is there any code in woof-ce to write the extended 'DISTRO_SPECS' in 'initrd.gz' and '/etc/DISTRO_SPECS' in 'puppy....sfs'?

Will the code work?
Probably, but we've always used the filename definititions in DISTRO_SPECS in the past.

Why is this a problem now?
Do we need to be creating an ndrv?
I don't understand why we would, but then I don't know what it is, and under what situation it is needed.
There was a suggestion that it needs to be loaded automatically because the user would not have enough English to navigate to load it as an extra-sfs. If that is so, then isn't their lack of English going to cause lots more significant problems?
If this is supposed to be for a translator, then I would have thought that they would have to know 2 languages, 1 in which Puppy is already available, and 1 in which it is not. So why would they not be able to mount this as an extra sfs in the Puppy running in the already translated language that they know?

Reply:
Traditionally, locales are left out in Puppy, so it's English-only. I want to add support for building a Puppy with locales, but make them optional because some users don't need them, hence the SFS. But it needs to be loaded by default, because that's what makes non-English speaking users able to change the language.

Reply:
@dimkr,
Is there woof-ce code to produce DISTRO_SPECS files with the "ndrv" information?

Reply:
Yes, I changed the relevant parts in 3builddistro (please correct me if I'm wrong).